src/cocotb_tools/runner.py

```diff
@@ -302,6 +302,7 @@ def test(
         gpi_interfaces: Optional[List[str]] = None,
         testcase: Optional[Union[str, Sequence[str]]] = None,
         seed: Optional[Union[str, int]] = None,
+        elab_args: Sequence[str] = [],
```

Comment:
Why would this be in the test call? Isn't elaboration only done once per *build*?

Reply:
Maybe not since other sims allow users pass new generics which re-elabs per test.